engine.py

```diff
@@ -52,24 +52,82 @@ def get_pawn_moves(self, r, c, moves):
 			return parameter(s):
 			None
 		"""
+		
+		def pawn_attack(pawn_type):
```

Comment:
while testing, it appears that pawn_attack() always defaults to a left capture when it has both right and left pieces to capture (both light and dark pawns show this problem)